server/app/services/program/ProgramService.java

```diff
@@ -622,22 +630,22 @@ private void clearCommonIntakeForm() {
    * actually update any programs.
    *
    * @param displayName a name for this program
-   * @param displayDescription the description of what the program provides
+   * @param shortDescription a short description (<100 characters) of what the program provides
    * @param externalLink A link to an external page containing additional program details
    * @param displayMode The display mode for the program
    * @param tiGroups The List of TiOrgs who have visibility to program in SELECT_TI display mode
    */
   public ImmutableSet<CiviFormError> validateProgramDataForUpdate(
       String displayName,
-      String displayDescription,
+      String shortDescription,
```

Comment:
short description is required and long description (aka the former description field) is now optional, so we no longer need to pass it through validation.